packages/helm-charts/blockscout/values.yaml

```diff
@@ -189,3 +189,10 @@ blockscout:
     image:
       repository: gcr.io/celo-testnet/celo-monorepo
       tag: metadata-crawler-77a392216d4927e85ce4b683508fc0539aa92a34
+nfs-server-provisioner:
+  persistence:
+    enabled: true
+    storageClass: "ssd"
+    size: 20Gi
```

Comment:
This is used for `PVC Storage Request for config volume`. I'd say probably 1 or 2 Gb would be much than enough.